### ISSUE TITLE
[FEATURE] Feeds Tags

### ISSUE THREAD

**Issue by kabir0x23** [thecyberworld/TheCyberHUB#449]
### Detailed description

Add a tags feature on the right side of the feeds, and get the reference from the explore section.
![image](https://github.com/thecyberworld/TheCyberHUB/assets/44284877/5ead8224-c6c5-458c-ad2d-a5ba8169163d)

### Context

This will improve the user performance

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vinayakhegde1** [comment]
Hello, @kabir0x23  I was trying to run the project locally to find some issues but I've been getting ERR_BLOCKED_BY_CLIENT is this because I'm using Brave browser?

**Comment by kabir0x23** [comment]
> Hello, @kabir0x23 I was trying to run the project locally to find some issues but I've been getting ERR_BLOCKED_BY_CLIENT is this because I'm using Brave browser?

can u share the screenshot?

**Comment by vinayakhegde1** [comment]
![image](https://github.com/thecyberworld/TheCyberHUB/assets/73926849/850d974b-2784-45a7-9b8e-ccc198c49bd3)

**Comment by kabir0x23** [comment]
this is something else

remove node modules and try again

**Comment by vinayakhegde1** [comment]
okay ill try to install node again let me check

**Comment by kabir0x23** [comment]
> okay ill try to install node again let me check
not node again, remove node_modules fron the profject and do npm ci to install packages agian
![image](https://github.com/thecyberworld/TheCyberHUB/assets/44284877/a369fe5a-d02e-4dfa-b186-bd67c155f77b)

**Comment by vinayakhegde1** [comment]
@kabir0x23 okay got it

**Comment by vinayakhegde1** [comment]
when i runned it on chrom it worked properly but with brave it's same
![image](https://github.com/thecyberworld/TheCyberHUB/assets/73926849/8e903a1b-feca-4c7c-9939-ac2fc4e99fd7)

**Comment by kabir0x23** [comment]
> when i runned it on chrome it worked properly but with brave it's showing the above error ![image](https://user-images.githubusercontent.com/73926849/271842331-8e903a1b-feca-4c7c-9939-ac2fc4e99fd7.png)

yeah have seen that the issue is only comming with privacy policy page in brave, will figure it out later

**Comment by vinayakhegde1** [comment]
@kabir0x23 should i create a issue if anybody knows how to slove it?

**Comment by kabir0x23** [comment]
> @kabir0x23 should i create a issue if anybody knows how to slove it?

yes u can, and try to add proper isse with example, that with this file it is it is giving issues in this  brosweer in local host k

**Comment by vinayakhegde1** [comment]
okay ill make new issue in detail

**Comment by Pratikdate** [comment]
Assign me i should work upon it

**Comment by vinayakhegde1** [comment]
@kabir0x23 i think you assigned me on this issue

**Comment by kabir0x23** [comment]
> @kabir0x23 i think you assigned me on this issue

Already done

**Comment by kabir0x23** [comment]
> Assign me i should work upon it

Hey u can checkout other issues or can create new one...

**Comment by vinayakhegde1** [comment]
@kabir0x23 assign to @Pratikdate  i'm actually working on a different issue which i will add by myself if its ok with you

**Comment by kabir0x23** [comment]
> @kabir0x23 assign to @Pratikdate i'm actually working on a different issue which i will add by myself if its ok with you

yeah no issues

**Comment by Pratikdate** [comment]
Thaks

**Comment by kabir0x23** [comment]
@Pratikdate hey are you working on this issue?

**Comment by 31Pratishtha** [comment]
Hey there, community ! I'd like to try work on this issue.

**Comment by kabir0x23** [comment]
> Hey there, community ! I'd like to try work on this issue.

Okay, great

**Comment by 31Pratishtha** [comment]
@kabir0x23 Do you think it should be done like this? Also, I wanted to confirm, clicking on a tag should render the posts related to that tag, right?

<img width="956" alt="image" src="https://github.com/thecyberworld/TheCyberHUB/assets/96373767/e60f9ee8-6116-4515-9f81-c30f5b7192e8">

**Comment by kabir0x23** [comment]
yes right, 

make sure to go through the installation guide as well, u are on main api, u have to rename example.env to .env
https://github.com/thecyberworld/TheCyberHUB/tree/dev?tab=readme-ov-file#-installation-guidelines

if u need any help let us know

thank you.

**Comment by 31Pratishtha** [comment]
Okay thanks. I'll look further into it.

**Comment by kabir0x23** [comment]
> Okay thanks. I'll look further into it.

and i was correct, u are also on wrong branch, make sure to copy your changes to somewhere
we are working on dev branch and u are now in main branch,  this one is the second issue 

all of the issues will be resolved via, 👉 [Installation-guide](https://github.com/thecyberworld/TheCyberHUB/tree/dev?tab=readme-ov-file#-installation-guidelines) 👈

**Comment by ParasAnmol** [comment]
Assign me
